Automattic/jetpack
# VideoPress: Add video preview on hover options to poster and preview panel

Fixes #29719

## Proposed changes:

* Adds the options to the panel, persisting its values, but does not use them yet

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add a Videopress video block and upload/select a video
* Go to the `Poster and preview` panel
* Check that a new toggle for `Video preview on hover` is present
* Check that, when toggled, the starting point and loop duration are editable
* Check the toggle, change some values and save the post
* Refresh the page
* Check that the values were persisted.

https://user-images.githubusercontent.com/8486249/228621739-baecf426-0e7f-4336-9c1e-d951ca8ea56a.mp4
